frontend/ui/src/pages/Channels/ChannelsSources/ChannelsConnected.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {Channel} from 'httpclient';
+import styles from './ChannelsConnected.module.scss';
+
+type connectedChannelsProps = {
+  showConnectedChannels: boolean;
+  showSumOfChannels: number;
+  connected: string;
+  connectedChannel: Channel[];
+  placeholderImage?: JSX.Element;
+  extraChannel: boolean;
+  displayExtraChannel: number;
+  isConnected: string;
+  addAChannel: JSX.Element;
+  ignoreChannelId?: boolean;
+  ignoreSvgAvatar?: boolean;
+  displayFacebookImage?: boolean;
+};
+
+const ChannelsConnected = (props: connectedChannelsProps) => {
+  return (
+    <>
+      {props.showConnectedChannels && (
+        <>
+          <div className={styles.connectedContainer}>
+            <div className={styles.connectedSum}>
+              <p>
+                {props.showSumOfChannels} {props.connected}
+              </p>
+            </div>
+            <div className={styles.connectedChannelBox}>
+              <div className={styles.connectedChannel}>
+                {props.connectedChannel.map((channel: Channel) => {
+                  const channelName = channel.metadata.name;
+                  return (
+                    <>
+                      <li key={channel.sourceChannelId} className={styles.channelListEntry}>
+                        <div className={styles.connectedChannelData}>
+                          {channel.metadata.imageUrl && props.displayFacebookImage && (
+                            <img src={channel.metadata.imageUrl} alt={channelName} className={styles.facebookImage} />
+                          )}
+
+                          {!props.ignoreSvgAvatar && (
+                            <div className={styles.placeholderLogo}>{props.placeholderImage} </div>
+                          )}
+
+                          <div className={styles.connectedChannelName}>{channel.metadata.name}</div>
+
+                          {!props.ignoreChannelId && <div className={styles.channelId}>{channel.sourceChannelId}</div>}
+                        </div>
+                      </li>
+                    </>
+                  );
+                })}
+              </div>
+              <div className={styles.extraChannelButton}>
+                {props.extraChannel && (
+                  <button className={styles.extraChannel} title="Click for more details">
```

Comment:
why create a new button? we already have this one in our library https://components.airy.co/?path=/story/airy-components-cta-linkbutton--link-button-component